easybuild/easyconfigs/o/OpenCV/OpenCV-3.4.1-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,89 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenCV'
+version = '3.4.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://opencv.org/'
+description = """OpenCV (Open Source Computer Vision Library) is an open source computer vision
+ and machine learning software library. OpenCV was built to provide
+ a common infrastructure for computer vision applications and to accelerate
+ the use of machine perception in the commercial products."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = [
+    'https://github.com/Itseez/opencv/archive/',
+    # The Hash is version dependent! see 3rdparty/ippicv/downloader.cmake
+    'https://raw.githubusercontent.com/Itseez/opencv_3rdparty/81a676001ca8075ada498583e4166079e5744668/ippicv',
```

Comment:
@hajgato This commit should be updated too:

```
$ grep COMMIT opencv-3.4.1/3rdparty/ippicv/ippicv.cmake
  set(IPPICV_COMMIT "dfe3162c237af211e98b8960018b564bc209261d")
                 "https://raw.githubusercontent.com/opencv/opencv_3rdparty/${IPPICV_COMMIT}/ippicv/"
```

Reply:
@boegel oops, I overlooked this.